### PR TITLE
for review

### PR DESCRIPTION
**Pull request by botlfarm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,110 @@
...
+Update the package installer and your server
 ```
 apt update
 apt upgrade
 ```
-install docker-compose and nano.
+Install docker-compose and nano.
 ```
 apt install docker-compose
 apt install nano
```
Comment (erlarese) [inline README.md:31]:
```suggestion
apt install docker-compose nano
```

File: README.md [hunk 2]
```diff
@@ -1,57 +1,110 @@
...
 
-Im using a shared 1gb ram nano server running Ubuntu 20.4 LTS
+I'm using a shared 1gb ram nano server running Ubuntu 20.4 LTS. This guide from Linode helps walk through this process. https://www.linode.com/docs/guides/getting-started/
 
 ## Domain 
-Witout a domain the ssl wont work. 
+You need to have a domain name pointed to the linode server for ssl to work. 
 
-Point your domain to Linode server ip adress. This will vary depending on where your domain is registered. You will need to set your name servers to point to linode servers and add an a record for your domain to point to your linode ip adress. 
+Point your domain to Linode server IP address. This will vary depending on where your domain is registered. You will need to set your name servers to point to linode servers and add an 'a record' for your domain to point to your linode IP address. Linode has a nice write up to help with this. https://www.linode.com/docs/guides/dns-manager/
```
Comment (erlarese) [inline README.md:14]:
```suggestion
Point your domain name to your Linode server IP address. This process will vary depending on where your domain name is registered. You will need to set your name servers to point to your Linode servers and add a 'record' for your domain to point to your Linode IP address. Linode has a nice write up to help with this. https://www.linode.com/docs/guides/dns-manager/
```

File: README.md [hunk 3]
```diff
@@ -1,57 +1,110 @@
...
-## Prerecuasit for server
-Login to your server. Replace [your ip adress] with your ip adress from linode
+## Setup of server
+In PowerShell login to your server. Replace [your IP address] with your IP address assigned to you by Linode
 ```
-ssh root@[your ip adress]
+ssh root@[your IP address]
 ```
-update the package installer and your server
+You will need to enter the root password you assigned while creating the Linode server. Note:it won't look like your typing anything when entering your password, just type it and hit enter
```
Comment (erlarese) [inline README.md:21]:
```suggestion
You will need to enter the root password you assigned while creating the Linode server.
**Note:** it won't look like you're typing anything when entering your password, just type it and hit enter.
```

File: README.md [hunk 4]
```diff
@@ -1,57 +1,110 @@
...
 Create the proxy network
-
 ```
 docker network create nginx-proxy
 ```
+Create proxy folder, copy the docker-compose.yml from github
+```
+mkdir proxy
+cd proxy
+curl https://raw.githubusercontent.com/botlfarm/Installing_farmOS/main/proxy/docker-compose.yml -o docker-dompose.yml
```
Comment (erlarese) [inline README.md:45]:
```suggestion
curl https://raw.githubusercontent.com/botlfarm/Installing_farmOS/main/proxy/docker-compose.yml -o docker-compose.yml
```

File: README.md [hunk 5]
```diff
@@ -1,57 +1,110 @@
...
+You should see 2 docker containers running the nginx and the nginx-companion
 
-check for running containers
+## farmOS
+Create farmOS folder, copy the docker-compose.yml file form github
+```
+cd ..
+mkdir farmOS
+cd farmOS
+curl https://raw.githubusercontent.com/botlfarm/Installing_farmOS/main/farmOS/docker-compose.yml -o docker-dompose.yml
```
Comment (erlarese) [inline README.md:69]:
```suggestion
curl https://raw.githubusercontent.com/botlfarm/Installing_farmOS/main/farmOS/docker-compose.yml -o docker-compose.yml
```

**Review by erlarese — COMMENTED** [review]
Hi Nick, I added some small suggestions to fix some wording and simplify/fix a couple of the commands.  Overall I think this looks good to me, let me know if you wanted any other feedback.

File: README.md (hunk 1 above)
Comment (botlfarm) [reply README.md:31]: fixed

**Pull request merged**